src/lib/services/playground-service.ts

```diff
@@ -0,0 +1,124 @@
+import FeatureToggleService from './feature-toggle-service';
+import { SdkContextSchema } from 'lib/openapi/spec/sdk-context-schema';
+import { InMemStorageProvider, Unleash as UnleashClient } from 'unleash-client';
+import { IUnleashServices } from 'lib/types/services';
+import { ALL } from '../../lib/types/models/api-token';
+import { PlaygroundFeatureSchema } from 'lib/openapi/spec/playground-feature-schema';
+import { Logger } from '../logger';
+import { IUnleashConfig } from 'lib/types';
+import { FeatureConfigurationClient } from 'lib/types/stores/feature-strategies-store';
+import { Operator } from 'unleash-client/lib/strategy/strategy';
+import { once } from 'events';
+
+enum PayloadType {
+    STRING = 'string',
+}
+
+type NonEmptyList<T> = [T, ...T[]];
+
+export const offlineUnleashClient = async (
+    toggles: NonEmptyList<FeatureConfigurationClient>,
+    context: SdkContextSchema,
+    logError: (message: any, ...args: any[]) => void,
+): Promise<UnleashClient> => {
+    const client = new UnleashClient({
+        ...context,
+        appName: context.appName,
+        disableMetrics: true,
+        refreshInterval: 0,
+        url: 'not-needed',
+        storageProvider: new InMemStorageProvider(),
+        bootstrap: {
+            data: toggles.map((x) => ({
```

Comment:
Is there a way to avoid this mapping? If TS doesn't allow you, can we silence the warning since we know that this format matches?

Reply:
We probably could, but do we want to? What if we end up changing something later on? We could always do something like this
```ts
        bootstrap: {
            // @ts-expect-error
            data: features,
        },
```
or this
```ts
const mapFeaturesForBootstrap = (features: FeatureConfigurationClient[]) =>
    features as unknown as FeatureInterface[];
```

but I'm not sure we want to. I'll let you pull the trigger on this one.